src/js/store/orgs/actions.ts

```diff
@@ -53,15 +54,17 @@ export type OrgsAction =
   | CommitEvent
   | OrgRefresh;
 
-export const provisionOrg = (payload: Org): ThunkResult => (
-  dispatch,
-  getState,
-) => {
+export const provisionOrg = ({
+  model,
+  originating_user_id,
+}: {
+  model: Org;
+  originating_user_id: string | null;
+}): ThunkResult<OrgProvisioned> => (dispatch, getState) => {
   const state = getState();
-  const user = state.user;
   /* istanbul ignore else */
-  if (user && user.id === payload.owner) {
-    const task = selectTaskById(state, payload.task);
+  if (isCurrentUser(originating_user_id, state)) {
```

Comment:
@jgerigmeyer I noticed this line is being ignored here, but it may be a left over from previous changes. Want to confirm if we should follow suit and ignore the else branch, or go ahead and add coverage for users who are not the owner.

Reply:
@duggiemitchell I'm fine either way. I'd say the logic is pretty clear, so I'm okay ignoring some or all of these `else` branches. 🤷‍♂